views/js/chatControl.js

```diff
@@ -1,6 +1,20 @@
 let chatActiveRoom = "general";
 const chatRefreshTimeMs = 30000;
 
+const hideChat = () => {
+  document.getElementById("wrapper_grid").style.gridTemplateColumns = "100% 0";
+  document.getElementById("chat_window").style.display = "none";
+  document.getElementById("hide_chat_button").style.display = "none";
+  document.getElementById("show_chat_button").style.display = "block";
+}
+
+const showChat = () => {
+  document.getElementById("wrapper_grid").style.gridTemplateColumns = "80% auto";
```

Comment:
It looks like the current ratio is `75% auto` in the `style.css` - we should standardize this.

Reply:
Thanks, I just set it to `null` to inherit the value defined originally in the css file. https://github.com/knative-portability/Kubercade/pull/28/commits/4010fac4a8edd449a735a6203e28db3781c3c44e